Controllers/NoteController.cs

```diff
@@ -0,0 +1,53 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Taskmanager.Controllers.InputModels;
+using Taskmanager.Controllers.ViewModels;
+using Taskmanager.Data.Entities;
+using Taskmanager.Services.Interfaces;
+
+namespace Taskmanager.Controllers
+{
+    [ApiController]
+    [Route("api/notes")]
+    public class NoteController : ControllerBase
+    {
+        private readonly INoteService _noteService;
+
+        public NoteController(INoteService noteService)
+        {
+            _noteService = noteService;
+        }
+
+        [HttpGet("{userId}/{todolistId}/{todoitemId}")]
+        public async Task<ActionResult<NoteViewModel>> GetOneByIdAsync([FromRoute] int userId, [FromRoute] int todolistId, [FromRoute] int todoitemId)
+        {
+            Note requiredNote = await _noteService.GetOneByIdAsync(userId, todolistId, todoitemId);
+
+            return NoteViewModel.MapNote(requiredNote);
+        }
+
+        [HttpPost("{userId}/{todolistId}/{todoitemId}")]
```

Comment:
Same here, you just need todoitemId as an FK to add new note, no need to provide other ids